### PR TITLE
Updated to Cordova 1.9.

### PR DESCRIPTION
**Pull request by wcwynn**
- Added example and updated the README.
- Added badge set/get methods.
- Added launch method.
- Added example code for optionally handling notifications in JS code.

Reviewer: @aogilvie

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
 # PLUGIN:
```
Comment (aogilvie) [inline README.md:5]: Update readme to reflect changes.

File: example/www/index.html [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+	// If you want to prevent dragging, uncomment this section
+	/*
+	function preventBehavior(e) 
+	{ 
+      e.preventDefault(); 
+    };
+	document.addEventListener("touchmove", preventBehavior, false);
+	*/
+
```
Comment (aogilvie) [inline example/www/index.html:47]: add new "receivedLocalNotification" event if the developer want to get all local notifications.

File: README.md (hunk 1 above)
Comment (wcwynn) [inline README.md:5]: Done.

File: example/www/index.html (hunk 2 above)
Comment (wcwynn) [inline example/www/index.html:47]: Done.

**Comment by wcwynn** [comment]
I updated the pull request.  I think things should be a bit cleaner/simpler now but if there are other ideas how to make cleaner let me know.

Unfortunately, iOS doesn't have a built-in UIApplication<XXX>Notification for observing when a local/remote notification occurs so we can't remove everything from the app delegate.  I did try to trim it down to the bare minimum required code.  I suspect there is such a notification internally in iOS, but it seems Apple has not made it a public one yet.  If you know of a way to get rid of the app delegate code, let me know and I'll remove it.

**Comment by aogilvie** [comment]
Awesome!

**Pull request merged**